server/config/passport.ts

```diff
@@ -27,10 +27,19 @@ passportUse(new LocalStrategy({
         .select('username email hashed_password salt')
         .exec((err, user) => {
             if (err) return done(err);
+            let errorObject = { message: 'wrong username or password' };
             if (!user || !user.authenticate(password)) {
-                return done(null, false, { message: 'wrong username or password' });
+                return done(null, false, errorObject);
```

Comment:
Do you really have to use the same function (`done`) with *several kind of arguments*? d: It looks like a by-hand pattern matching to me.